lib/index.js

```diff
@@ -670,7 +670,8 @@ exports.bindEmitter = function (em) {
       // last event has been setup when the http instrumentation binds the
       // events but there must be CLS context.
       if (exports.tracing || (clsCheck() && (em.headers && em.socket))) {
-        return exports.requestStore.bindEmitter(em)
+        exports.requestStore.bindEmitter(em)
```

Comment:
Question: Is this cosmetic or does it change the timing of when the bind happens?

Reply:
requestStore.bindEmitter() does not return the bound emitter; it just wraps it, so the return value was incorrect previously.